lib/galaxy/managers/workflows.py

```diff
@@ -77,7 +77,7 @@ def get_stored_workflow(self, trans, workflow_id, by_stored_id=True):
         else:
             workflow_id = decode_id(self.app, workflow_id)
             workflow_query = trans.sa_session.query(trans.app.model.StoredWorkflow).filter(and_(
-                trans.app.model.StoredWorkflow.latest_workflow_id == trans.app.model.Workflow.id,
+                trans.app.model.StoredWorkflow.id == trans.app.model.Workflow.stored_workflow_id,
```

Comment:
Shouldn't be the same also on line 70?

Reply:
Ahhh - I was wondering what in the world I was thinking when I added that. Makes sense where I got it from now, thanks @nsoranzo !